src/foraging_gui/Foraging.py

```diff
@@ -1490,7 +1497,6 @@ def _Task(self):
             self._AdvancedBlockAuto() # Update states of SwitchThr and PointsInARow
             
             self.BlockMinReward.setEnabled(True)
```

Comment:
BlockMinReward was enabled here when you switch to coupled task. I don't understand why it didn't work for you.